DXRVanilla/DeusEx/Classes/Player.uc

```diff
@@ -1669,23 +1669,33 @@ exec function PlayerRot()
     ClientMessage("Player rotation: (" $ Rotation.pitch $ ", " $ Rotation.yaw $ ", " $ Rotation.roll $ ")");
 }
 
-exec function ShowRefused()
+exec function LootActions()
 {
-    local string refusals, msg;
+    local string lootActions, msg;
     local int idx;
 
-    refusals = class'DataStorage'.static.GetObj(GetDXR()).GetConfigKey("item_refusals");
+    lootActions = class'DataStorage'.static.GetObj(GetDXR()).GetConfigKey("loot_actions");
 
     // basically just adds a space after every comma
-    while (Len(refusals) > 1) {
-        refusals = Right(refusals, Len(refusals) - 1);
-        idx = InStr(refusals, ",");
-        msg = msg $ Left(refusals, idx) $ ", ";
-        refusals = Right(refusals, Len(refusals) - idx);
+    lootActions = Mid(lootActions, 1);
+    while (lootActions != "") {
+        idx = InStr(lootActions, ",");
+        msg = msg $ Left(lootActions, idx) $ ", ";
+        lootActions = Mid(lootActions, idx + 1);
     }
     msg = Left(msg, Len(msg) - 2);
 
-    ClientMessage("Refused items: " $ msg);
+    ClientMessage("Loot actions: " $ msg);
+}
+
+function bool ConsumableWouldHelp(Inventory item) {
```

Comment:
not sure I like this single long return statement, maybe this is cleaner even though it's longer

```
function bool ConsumableWouldHelp(Inventory item) {
    if (health < default.health) {
        if (MedKit(item) != None || SoyFood(item) != None || Candybar(item) != None) {
            return true;
        }
        if (HealingItem(item) != None && HealingItem(item).health > 0) {
            return true;
        }
    }

    if (energy < default.energy) {
        if (BioElectricCell(item) != None) {
            return true;
        }
        if (HealingItem(item) != None && HealingItem(item).energy > 0) {
            return true;
        }
    }

    return false;
}
```

Reply:
I quite like the single boolean expression. But I'll defer to you.

Reply:
Ah, I see that I already have.